lib/fastlane/plugin/dynatrace/actions/dynatrace_action.rb

```diff
@@ -55,10 +55,18 @@ def self.run(params)
         end
 
         # iOS/tvOS workflow
+        unless OS.mac?
+          UI.user_error! "A macOS machine is required to process iOS symbols."
```

Comment:
```suggestion
          UI.user_error! "A macOS machine is required to process iOS or tvOS symbols."
```
Unrelated, but you're touching it 😆